### PR TITLE
fix(Header): Added position: relative; back to logo

### PR DESCRIPTION
**Pull request by dborysiewicz**
Added position: relative; back to logo to restrict the position: absolute; link.

### CODE REVIEW

File: react/Header/Header.less [hunk 1]
```diff
@@ -80,28 +80,32 @@
...
-  position: absolute;
-  top: 0;
-  text-indent: 100%;
-  overflow: hidden;
-  width: 100%;
-  height: 100%;
+.logoNote {
+  .standardText();
+  margin-left: 0.8rem;
+  margin-bottom: 0.35rem;
```
Comment (michaeltaranto) [inline react/Header/Header.less]: Can i ask what these numbers are? We dont really have/use magic numbers for spacing (i understand this is an aside to what you are fixing, but think it should be done as part of this tidy up)

Comment (kosanna) [inline react/Header/Header.less]: This is a proper code-review @michaeltaranto 👍 These are just adjusting the note to be aligned with the `seek` word in the logo, probably better to have it all just as an image asset all together 🤔

Comment (kosanna) [inline react/Header/Header.less]: Let's do the fix in a separate PR, closing this one
